### PR TITLE
Scripts/Spells: Suppression Aura should not hit stealthed units.

### LINKED ISSUE #11249: [BWL] Suppression aura should not hit the stealth rogue
### Current Behaviour

Suppression aura hit all the players.

### Expected Blizzlike Behaviour

Rogue in stealth shouldn't be hit by the suppression aura. 

### Source

https://youtu.be/uwD8QmXpwZU?t=9

### Steps to reproduce the problem

1. Go to suppression with a rogue
2. Go in stealth
3. You should hit by a suppression device

### Extra Notes

 It wasn't specify in the original report from #10834.

### AC rev. hash/commit

285504e

### Operating system

Windows 11

### Custom changes or Modules

_No response_

### PR DESCRIPTION
**Pull request by UltraNix**
Fixes #11249

## Issues Addressed:

- Closes #11249

## Tests Performed:

- Not tested.

## How to Test the Changes:

Go to suppression with a rogue
Go in stealth
You should not be hit by a suppression device.

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

**Comment by Lynethris** [comment]
Tested and work

**Comment by Nefertumm** [comment]
missing SpellScript header

**Comment by UltraNix** [comment]
@Nefertumm Done.

File: data/sql/updates/pending_db_world/rev_1648996451912542300.sql [hunk 1]
```diff
@@ -0,0 +1,5 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1648996451912542300');
+
+DELETE FROM `spell_script_names` WHERE `spell_id`=22247;
+INSERT INTO `spell_script_names` VALUES
```
Comment (Nefertumm) [inline data/sql/updates/pending_db_world/rev_1648996451912542300.sql:4]:
```suggestion
INSERT INTO `spell_script_names` (`spell_id`, `ScriptName`) VALUES
```

**Review by Nefertumm — APPROVED** [review]

**Comment by acidmanifesto** [comment]
> Tested and work

Can you re-test this again, since the script loader was missing before you tested. I want to absolutely make sure this works.

**Comment by Lynethris** [comment]
> > Tested and work
> 
> Can you re-test this again, since the script loader was missing before you tested. I want to absolutely make sure this works.

Tested and it's work always.

**Comment by acidmanifesto** [comment]
> > > Tested and work
> > 
> > 
> > Can you re-test this again, since the script loader was missing before you tested. I want to absolutely make sure this works.
> 
> Tested and it's work always.

thank you for verifying.

**Pull request merged**